src/main/scala/au/com/agiledigital/toolform/command/generate/kubernetes/minikube/GenerateMinikubeCommand.scala

```diff
@@ -73,8 +74,9 @@ object GenerateMinikubeCommand extends YamlWriter {
         _ <- write(s"# Date: ${DateUtil.formattedDateString}")
         _ <- project.components.values.filter(shouldWriteService).toList.traverse_(writeService)
         _ <- project.resources.values.filter(shouldWriteService).toList.traverse_(writeService)
-        _ <- project.components.values.toList.traverse_((component) => writeDeployment(project.id, component))
-        _ <- project.resources.values.toList.traverse_((resource) => writeDeployment(project.id, resource))
+        _ <- project.resources.values.filter(isdiskResourceType).toList.traverse_((resource) => writeVolumeClaim(resource))
+        _ <- project.components.values.toList.traverse_((component) => writeDeployment(project, component))
+        _ <- project.resources.values.filter(isNotDiskResourceType).toList.traverse_((resource) => writeDeployment(project, resource))
```

Comment:
We can use `filterNot` here and avoid the duplicate definition

Reply:
Handy. Its been updated with that, thanks.